src/cript/api/exceptions.py

```diff
@@ -129,6 +130,35 @@ def __str__(self) -> str:
         return error_message
 
 
+class CRIPTDuplicateNameError(CRIPTAPISaveError):
```

Comment:
is this block supposed to be in this PR?

Reply:
It shows up, because branch is not up to date with `develop`.
But I based the trunk update on the new `develop`.